it/pom.xml

```diff
@@ -100,6 +100,15 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-failsafe-plugin</artifactId>
                         <version>3.0.0-M5</version>
+                        <executions>
+                            <execution>
+                                <id>integration-test</id>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
```

Comment:
Could you explain why is this needed? for me it looks like the change in `pull_request.yml` will bring back the universe test?

Reply:
This adds integration-test task into the verification task, it's a necessary change
P.S. as I remember universe tests worked even without this change though, but it's good to have it here

Reply:
Without this "integration-test" goal, the integration-tests were getting skipped with the "universe" profile id. It used to work before with the "it" profile id. Hence I had to add this goal specifically.